tests/TestCase/ORM/EntityTest.php

```diff
@@ -273,6 +275,27 @@ public function testGetNoGetters(): void
         $this->assertSame('bar', $entity->get('foo'));
     }
 
+    public function testRequirePresenceException(): void
+    {
+        $this->expectException(MissingPropertyException::class);
+        $this->expectExceptionMessage('Property `foo` does not exist for the entity `Cake\ORM\Entity`');
+
+        $entity = new Entity();
+        $entity->requireFieldPresence();
+        $entity->get('foo');
```

Comment:
I try to express intent with such strings. Helps the readability.

```suggestion
        $entity->get('not_present');
```